eftest/src/eftest/runner.clj

```diff
@@ -39,18 +40,26 @@
                         :var      v})))
       result)))
 
-(defn- run-in-parallel [fs]
-  (let [executor (java.util.concurrent.Executors/newFixedThreadPool
-                   (+ 2 (.availableProcessors (Runtime/getRuntime))))]
+(defn- ^Callable bound-callback [f]
+  (cast Callable (bound-fn* f)))
+
+(defn- ^ExecutorService threadpool-executor []
+  (Executors/newFixedThreadPool (+ 2 (.availableProcessors (Runtime/getRuntime)))))
+
+(defn- pcalls* [fs]
+  (let [executor (threadpool-executor)]
     (try
       (->> fs
-           (map #(.submit executor ^Runnable %))
-           doall
-           (map #(.get %))
-           dorun)
+         (map #(.submit executor (bound-callback %)))
```

Comment:
Looks like the indentation is off here. Can you tidy it up?

Reply:
sure, will do it